acceptance_process.md

```diff
@@ -3,39 +3,11 @@ The following is an approval process for PKT Network Steward funded projects. Fi
-    * **Note**: Counter-proposals are submitted without knowledge of whether the project which they are countering will in fact be accepted
-* **Step 2**: (1 week after step 1)
-  * NS evaluates each counter-proposal according to evaluation criteria and compares it to the relevant original proposal, any counter-proposal which is not significantly similar and a significant improvement over the original proposal is eliminated and the original proposal wins.
-    **Note**: If there are multiple counter-proposals, they are ranked and all are immediately eliminated except for the best one
-  * Original applicants to provide amendments to original proposals to make them better than counter-proposals.
-    **Note**: Amendments to original applications are submitted without knowledge of whether the counter-proposal will be considered better
-    **Note 2**: If the counter-proposal is not accepted then the original project the one that wins, not the amendment.
-* **Step 3**: (1 week after step 2)
-  * NS evaluates amended original proposals and compares them to counter-proposals, any amended proposal which is not significantly similar and a significant improvement over the relevant counter-proposal is eliminated and the counter-proposal wins.
-  * Counter-proposers provide amendments to their counter-proposals, if a winner has not been selected at this point, the process continues at step 2.
-
-Once all competing proposals have been eliminated, the remaining proposal is deemed accepted and is funded. This happens even if other proposals remain in the amendment loop.
-
-If all involved applicants, counter-applicants and the NS team are in agreement, the amendment loop can happen faster than a 1 week period, but in case of non-response, 1 week is the deadline after which a participant is deemed to have failed to file an amendment to their proposal or counter-proposal.
-
-## Example of process
-1. Deadline 1: Alice submits a proposal
-* Deadline 2:
-    * Bob and Charlie, both thinking they can do the same project better/cheaper, submit counter-proposals
-    * The NS team provides their ranking and indicates that Alice's proposal has not been eliminated
-* Deadline 3:
-    * Alice submits an amended proposal which takes into account some of the improvements in the counter-proposals of Bob and Charlie
-    * The NS indicates that Bob's proposal is better than Alice's, Charlie's proposal being not better than Bob's is immediately eliminated
-* Deadline 4:
-    * Bob submits an amended counter-proposal
-    * The NS indicates that Alice's amended proposal is better than Bob's counter-proposal
-* Deadline 5:
-    * Alice submits a second amendment of her initial proposal
-    * The NS indicates that Alice's first amendment of her proposal is better than bob's amended counter-proposal, thus Bob's counter-proposal is eliminated and Alice's first amendment of her proposal wins
+* **Step 1** (48 hours after step 0 deadline)
```

Comment:
Good point, this is not new though, people could submit similar proposals even with the counter-proposal process. The current band-aid solution is that I act as an air traffic controller for proposals because most of the applicants ask my advice on what they should work on. One reason why I'd like to bring in professional evaluators is because I would like to charge for my time consulting, but with me as an evaluator that is a more overt conflict than I am willing to accept - so currently I advise anyone who asks on a volunteer basis.

We should also consider that there is some "eggs in one basket" risk associated with accepting too many proposals from the same applicant, so I would be quite favorable to a mutual exclusion deduction which worked for both purposes, however I'm not sure we should block this round of changes to get it.